code/drasil-printers/lib/Language/Drasil/Markdown/Book.hs

```diff
@@ -0,0 +1,30 @@
+-- | Defines functions to create .toml config files for mdBook.
+module Language.Drasil.Markdown.Book where
+
+import Text.PrettyPrint (Doc, text, vcat, (<+>))
+import Data.Map (empty)
+
+import Language.Drasil.Markdown.Print (pSpec)
+import Language.Drasil.Printing.PrintingInformation (PrintingInformation)
+import Language.Drasil.Printing.Import.Sentence (spec)
+
+import Language.Drasil hiding (Expr)
+
+-- | Generates the .toml config file for mdBook.
+makeBook :: Document -> PrintingInformation -> Doc  
+makeBook (Document t _ _ _) sm = vcat [
```

Comment:
Is it worth creating a record data type that carries the [configuration (according to their spec)](https://rust-lang.github.io/mdBook/format/configuration/general.html) before building the TOML?

Reply:
As of now, all the TOMLs will be identical, apart from the title. I kept the TOML quite simple, only defining the absolute minimum needed for the book to run as desired. I don't anticipate a need to use the other configuration options.